node-notes-cli/app.js

```diff
@@ -0,0 +1,47 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+function read() {
+  for (let i = 1; i <= data.nextId; i++) {
+    if (data.notes[i] !== undefined) {
+      console.log(`${i}: ${data.notes[i]}`);
+    }
+  }
```

Comment:
You can loop through each key of `data.notes` with a `for...in` loop. The `data.nextId` is just used during creation of new notes.